01_fib/fib.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+)
+
+func fib(n int) int {
+	f := make([]int, n+1, n+2)
```

Comment:
Also, you don't strictly need a slice here. Only the two most recent numbers are required to compute the next number. If you do so, it would be handy to return the entire slice rather than just the final number.